### DISCUSSION TITLE
浅浅拍个荷花

### DISCUSSION THREAD

**Discussion by SCJWY** [share121/inter-knot#2340] [其他]
![Uploading 微信图片_20240816085956.jpg…]()

**Comment by 996zhi** [comment]
兄弟你图呢？

**Reply by SCJWY** [reply]
没传上，重发了，一样的标题

**Comment by cpu644** [comment]
你的图提示还在上传。你要等上传完了才能发。
![image](https://github.com/user-attachments/assets/a79a6bd8-afc4-473a-8e32-fa3d9c158085)

**Comment by cpu644** [comment]
就像这样。
![image](https://github.com/user-attachments/assets/8e338d8f-15fc-439d-ac0d-168321329efe)

**Reply by SCJWY** [reply]
okok，了解了
